pylistenbrainz/client.py

```diff
@@ -362,3 +362,38 @@ def get_user_releases(self, username, count=25, offset=0, time_range='all_time')
         :rtype: dict
         """
         return self._get_user_entity(username, 'releases', count, offset, time_range)
+
+
+    def get_user_recommendation_recordings(self, username, artist_type='top', count=25, offset=0):
+        """ Get recommended recordings for a user.
+
+        :param username: the username of the user whose recommended tracks are to be fetched.
+        :type username: str
+
+        :param artist_type: The type of filtering applied to the recommended tracks. Only
+                            'top' is currently supported.
+        :type artist_type: str
+
+        :param count: the number of recordings to fetch, defaults to 25, maximum is 100.
+        :type count: int, optional
+
+        :param offset: the number of releases to skip from the beginning, for pagination, defaults to 0.
+        :type offset: int, optional
+
+        :return: the recommended recordings as other data returned by the API
+        :rtype: dict
+        """
+
+        assert artist_type == 'top'
```

Comment:
The endpoint takes artist_type = "similar" as well. We should allow for that too.

Reply:
Is that live yet?

Reply:
I see it is. Nice!